### PR TITLE
Merge wc_reference_data_2024.4.yaml and wc_reference_data.yaml

### PR DESCRIPTION
**Pull request by alexsu52**
### Changes
Merge wc_reference_data_2024.4.yaml and wc_reference_data.yaml

### Reason for changes
Simplify maintenance due to the migration to OV 2024.4 

### Related tickets
N/A

### Tests
post_training_weight_compression/198/

### CODE REVIEW

**Review by l-bat — APPROVED** [review]

File: tests/post_training/data/wc_reference_data.yaml [hunk 1]
```diff
@@ -7,35 +7,36 @@ tinyllama_data_aware_backend_OV:
...
   num_int8: 312
 tinyllama_data_aware_gptq_backend_OV:
-  metric_value: 0.87134
+  metric_value: 0.86521
   num_int4: 94
   num_int8: 124
   metrics_xfail_reason: "Issue-148819"
 tinyllama_scale_estimation_per_channel_backend_OV:
-  metric_value: 0.80873
+  metric_value: 0.80853
```
Comment (ljaljushkin) [inline tests/post_training/data/wc_reference_data.yaml:31]: Shouldn't it be marked by xfail, since 197 build failed with 0.80873 reference?

Comment (alexsu52) [reply tests/post_training/data/wc_reference_data.yaml:31]: In agreement with @l-bat, I updated the metrics. @l-bat, please, confirm that the result is reproducible on different hardware.

Comment (l-bat) [reply tests/post_training/data/wc_reference_data.yaml:31]: This test has become unstable on openvino-nightly again (details at ticket: 152627)

Comment (alexsu52) [reply tests/post_training/data/wc_reference_data.yaml:31]: I do not update openvino-nightly metrics.

**Review by ljaljushkin — APPROVED** [review]

**Pull request merged**